website1/css/main.css

```diff
@@ -0,0 +1,191 @@
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+
+body {
+    line-height: 1;
+}
+
+ol, ul {
+    list-style: none;
+}
+
+blockquote, q {
+    quotes: none;
+}
+
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+
+.header {
+    position: absolute;
+    width: 1440px;
+    height: 81px;
+    left: 0;
+    top: 0;
+    align-items: center;
+
+    background: rgba(255, 255, 255, 0.6);
+    backdrop-filter: blur(25px);
+}
+
+.logo_company {
+    width: 151px;
+    height: 81px;
+    margin-left: 75px;
+}
+
+.name_company {
+    position: absolute;
+    width: 286px;
+    height: 29px;
+    left: 226px;
+    top: 26px;
+
+    /*font-family: 'NEXT ART';*/
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 159.02%;
+    color: #4F4A45;
+
+}
+
+.menu a {
+    color: #4F4A45;
+    width: auto;
+    text-decoration: none;
+
+}
+
+.menu li {
+    margin-right: 20px;
+}
+
+.menu {
+    position: absolute;
+    /*width: 149px;*/
+    height: 29px;
+    left: 565px;
+    top: 26px;
+
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 159.02%;
+
+    display: flex;
+}
+
+.nav {
+    display: flex;
+    /*text-align: center;*/
+
+    /*    position: absolute;*/
+    /*    width: 1440px;*/
+    /*    height: 81px;*/
+    /*    left: 0px;*/
+    /*    top: 0px;*/
+
+    /*    background: rgba(255, 255, 255, 0.6);*/
+    /*    backdrop-filter: blur(25px);*/
+}
+
+.language {
+    position: absolute;
+    width: 23px;
+    height: 29px;
+    left: 1122px;
+    top: 26px;
+
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 159.02%;
+    color: #FFC300;
+    text-transform: uppercase;
+    text-decoration: none;
+}
+
+.btn_call {
+    display: flex;
+
+
+    box-sizing: border-box;
+
+    position: absolute;
+    width: 190px;
+    height: 48px;
+    left: 1175px;
+    top: 17px;
+
+    border: 2px solid #FFC300;
+
+}
+
+.btn_txt {
+    text-decoration: none;
+    position: absolute;
+    width: 162px;
+    height: 29px;
+
+    left: 14px;
+    top: 9px;
+
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 159.02%;
+    /* identical to box height, or 29px */
+
+
+    color: #4F4A45;
+}
+
+.btn_call:hover {
+    background: #FFC300;
+}
+
+.btn_txt:hover {
+    color: #fff;
+}
```

Comment:
The color of the text should change when you `hover` over the button, not the text.